stage.sh

```diff
@@ -12,7 +12,7 @@ BUCKET=$1
 S3PATH=$2
 
 # remove this line if you want to keep the objects private in your S3 bucket
```

Comment:
if we make it commented by default, I would change the comment to "uncomment this line [...]"

Reply:
This change is overwritten and will not pass into result commit.